spec/features/operator/triggers/triggers_Install_test.go

```diff
@@ -0,0 +1,14 @@
+package triggers
+
+import (
```

Comment:
could you please keep the filenames to lowercase and just call it `install_test.go` since it is already inside `triggers` directory?

Reply:
done